### PR TITLE
v1 API  update_time, name, batch_restore_version increase coverage

### PR DESCRIPTION
**Pull request by grusev**
#### What does this implement or fix?

update_time - no coverage in tests. A test for existing/non-existing symbol + versions valid/invalid
batch_restore_version - Good coverage exists. Missing: Test with as_of negative number and a snapshot, unuqual lists
name() - not covered - we could add an a small coverage in a test

#### Checklist

<details>
  <summary>
   Checklist for code changes...
  </summary>
 
 - [ ] Have you updated the relevant docstrings, documentation and copyright notice?
 - [ ] Is this contribution tested against [all ArcticDB's features](../docs/mkdocs/docs/technical/contributing.md)?
 - [ ] Do all exceptions introduced raise appropriate [error messages](https://docs.arcticdb.io/error_messages/)?
 - [ ] Are API changes highlighted in the PR description?
 - [ ] Is the PR labelled as enhancement or bug so it appears in autogenerated release notes?
</details>

### CODE REVIEW

File: python/tests/integration/arcticdb/version_store/test_basic_version_store.py [hunk 1]
```diff
@@ -1144,6 +1145,37 @@ def test_update_times(basic_store):
...
+    lib.write("sym1", nparr)
+    lib.write("sym1", series)
+    lib.snapshot("snap")
+    lib.write("sym1", df)
+
+    # Negative number for as_of works as expected
+    assert lib.update_time("sym1") == lib.update_time("sym1", -1) == lib.update_time("sym1", 2)
+    # Snapshots are accepted
+    assert lib.update_time("sym1", 1) == lib.update_time("sym1", -2) == lib.update_time("sym1", "snap")
+    # All types of supported data have update_time
```
Comment (vasil-pashov) [inline python/tests/integration/arcticdb/version_store/test_basic_version_store.py]: I think the comment is a bit confusing. I get the lines 1161, 1263 and 1165 combined test all types of input data but the comment makes it look like line 1165 tests all types of supported data types.

Comment (grusev) [reply python/tests/integration/arcticdb/version_store/test_basic_version_store.py]: fixed
